service/src/integrationTest/kotlin/fi/espoo/evaka/shared/controllers/ScheduledOperationControllerIntegrationTest.kt

```diff
@@ -242,16 +251,74 @@ class ScheduledOperationControllerIntegrationTest : FullApplicationTest() {
         assertEquals(ApplicationStatus.SENT, applicationStatus)
     }
 
+    @Test
+    fun `transfer application cancelling cleans up placement plans and decision drafts`() {
+        val preferredStartDate = LocalDate.now()
```

Comment:
Is it possible to avoid `LocalDate.now()` here or do we depend on today's date in some of the other test data already?

Reply:
The behavior depends on the current date, there's no dependency on other temporal data.